### DISCUSSION TITLE
Guess country

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#535] [Galaxy Trivia] [answered]
⭕👨
🔔🏋️
✌️®️🗝️
🛥️🦢🅰️

**Comment by ParanoidUser** [comment]
@ErKiran, @AlienegraGeek, @NobleOsinachi any ideas? 😶‍🌫️

**Comment by peterbaccaro** [comment, marked as answer]
OMAN
BELGIUM
TURKEY
BOTSWANA

**Comment by ErKiran** [comment]
let me try 
OMAN
BELGIUM
Switzerland
DENMARK

**Reply by ParanoidUser** [reply]
Almost there! Don't worry, you're still a winner in my book. 😄

**Comment by AlienegraGeek** [comment]
interesting...
OMAN
BELGIUM
TURKEY
CANADA

**Reply by ParanoidUser** [reply]
So close! You missed the last one, but you're still in the game! Keep playing and see what happens. 😉
